mirrord/layer/src/socket/id.rs

```diff
@@ -0,0 +1,25 @@
+//! The types in this module should **NOT** be constructed directly.
```

Comment:
You don't have to put it in a different file to have it as a different module, if you think it belongs in the same file - you could just wrap it in `mod socket_id { }` .

Reply:
`socket.rs` already has too much stuff imo.

Would you prefer to keep it as just a `mod` inside the file though?

Reply:
No, up to you :)